javaparser-core/src/main/java/com/github/javaparser/ast/comments/Comment.java

```diff
@@ -61,7 +63,7 @@ public final String getContent() {
      */
     public Comment setContent(String content) {
         notifyPropertyChange(ObservableProperty.CONTENT, this.content, content);
-        this.content = content;
+        this.content = assertNotNull(content);
```

Comment:
I think I agree, but that makes the location of just about every assertNotNull wrong :-O New issue?